test/token-verification.test.js

```diff
@@ -288,3 +288,52 @@ describe('jwt-verification', function () {
     });
   });
 });
+describe('access_token validation', function() {
+  it('should handle empty/null access tokens', function(done) {
```

Comment:
Change to:
```
should throw on empty/null access tokens
```

btw. What about undefined? I don't think that after you parse an AS response you get "explicit nulls". Would it be the same?

Reply:
fixed. added all test cases